src/app/loading.tsx

```diff
@@ -1,6 +1,6 @@
 export default function Loader() {
   return (
-    <div className="text-center">
+    <div className="h-screen w-full flex justify-center items-center">
```

Comment:
it would be better if you apply max height property instead of height property

Reply:
min-h-screen!? coz max-h will not work even with w-full